frontend/src/components/base/Table/Table.test.tsx

```diff
@@ -1,142 +1,192 @@
 import React from 'react';
+import { screen } from '@testing-library/react';
+import { ColumnDef } from '@tanstack/react-table';
+
 import { render } from '../../../test-utils';
 import { Table } from './Table';
 
-const headContent = <div>This is the head content</div>;
-const tableHeads = [
-  { title: 'Title 1', key: 1 },
-  { title: 'Title 2', key: 2 },
-  { title: 'Title 3', key: 3 },
-  { title: 'Title 4', key: 4 },
-  { title: 'Title 5', key: 5 },
-];
-const tableRows = [
+const header = <div>This is the head content</div>;
+
+type Person = {
+  firstName: string;
+  lastName: string;
+  age: number;
+  visits: number;
+  progress: number;
+  status: 'relationship' | 'complicated' | 'single';
+  subRows?: Person[];
+};
```

Comment:
I know this is just a test, but maybe we can use something more adequate like BlockHeight, BlockHash etc ;) 😅